### DISCUSSION TITLE
Problem with MATLAB +MTEX und libmvec.so.1

### DISCUSSION THREAD

**Discussion by Niknobel** [mtex-toolbox/mtex#937] [Ask Anything]
Hello i have installed mtex 5.6.1 new version but again the following error appear with libmvec.so.1. What should i do?

TEX: Error running NFFT!

   The original error message was:
   Invalid MEX-file 
'/p/bv/mikro/home/nobel/MATLAB/mtex-5.6.1/extern/nfft/nfsftmex.mexa64': 
libmvec.so.1: cannot open shared object file: No such file or directory

  I did not get NFFT working. This restricts the functionality of MTEX.
  To overcome this restriction you may need to compile the NFFT your own.
  Please have a look at 
github.com/mtex-toolbox/mtex/blob/develop/extern/nfft_openMP/readme.md

kind regards
Nikita

**Comment by kilir** [comment]
Hi Nikita,

have a look here: https://github.com/mtex-toolbox/mtex/issues/402
Your glibc is most likely too old (<2.22, released 2015). Could that be the case?

Cheers,
Rüdiger

**Reply by Niknobel** [reply]
Hi, 
I try following commands
on https://github.com/mtex-toolbox/mtex/blob/develop/extern/nfft_openMP/readme.md
Copy the NFFT Binaries into MTEX

Copy the following files into this directory mtex/extern/nfft_openMP

nfft/matlab/nfsoft/nfsoftmex.mex* 
nfft/matlab/nfsft/nfsftmex.mex* 

And file  nfsftmex.mexa64 points on  .libs/libnfsft.mexa64 but this file does not exist. What can i do?

thx kind regards
Nikita

**Reply by kilir** [reply]
What do you mean by does not exist? When building using the options from the readme.md I get the following:
```
[76.62 ks][rk@LL]:~/builds/nfft-3.5.2 $ls -la matlab/nfsoft/.libs/
total 440
drwxr-xr-x 2 rk rk   4096  7. Apr 21:15 .
drwxr-xr-x 4 rk rk   4096  7. Apr 21:15 ..
lrwxrwxrwx 1 rk rk     15  7. Apr 21:15 libnfsoft.la -> ../libnfsoft.la
-rw-r--r-- 1 rk rk   1030  7. Apr 21:15 libnfsoft.lai
-rw-r--r-- 1 rk rk  19184  7. Apr 21:15 libnfsoft_la-nfsoftmex.o
-rwxr-xr-x 1 rk rk 415824  7. Apr 21:15 libnfsoft.mexa64

[76.63 ks][rk@LL]:~/builds/nfft-3.5.2 $ls -la matlab/nfsft/.libs/
total 452
drwxr-xr-x 2 rk rk   4096  7. Apr 21:15 .
drwxr-xr-x 5 rk rk   4096  7. Apr 21:15 ..
lrwxrwxrwx 1 rk rk     14  7. Apr 21:15 libnfsft.la -> ../libnfsft.la
-rw-r--r-- 1 rk rk   1024  7. Apr 21:15 libnfsft.lai
-rw-r--r-- 1 rk rk  27328  7. Apr 21:15 libnfsft_la-nfsftmex.o
-rwxr-xr-x 1 rk rk 420176  7. Apr 21:15 libnfsft.mexa64
```
What does `ls -la matlab/nfsft/.libs/` tell you?
